lib/dice.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+import re
+
+class Dice():
+
```

Comment:
This probably doesn't need to be a class.

Reply:
Or it could be a class extended out if we want to represent many types of dice as a type in the future. Probably don't need that yet though